lib/tasks/data_load_and_save.rake

```diff
@@ -204,7 +204,8 @@ namespace :db do
                     data[c.name] = user_id
                   end
                 end
-                ActiveRecord::Base.connection.execute "INSERT INTO #{tbl} (#{data.keys.join(",")}) VALUES (#{data.values.collect { |value| ActiveRecord::Base.connection.quote(value) }.join(",")})", 'Fixture Insert'
+                column_keys = data.keys.join(",").sub(",key",",`key`")
```

Comment:
This will break things if there's a column name that starts with `key`. Currently there's not, and since this deals with grade span expectations anyway, I'm ok with just leaving it as is.